test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/MapTest.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.jsinterop
+
+import org.junit.Assert._
+import org.junit.Assume._
+import org.junit.{BeforeClass, Test}
+import org.scalajs.testsuite.utils.AssertThrows._
+import org.scalajs.testsuite.utils.Platform
+
+import scala.scalajs.js
+
+object MapTest {
+  @BeforeClass
+  def assumeRuntimeSupportsMap(): Unit = {
+    assumeTrue("Assume Map exists in Global",
+        js.typeOf(js.constructorOf[js.Map[_, _]]) != "undefined")
```

Comment:
- Move this to `Platfrom` for consistency with Symbols and TypedArrays?
- This needs to be tested without runtime check if `assumeES2015` is true.

Reply:
Replaced with `LinkingInfo.assumeES6` as suggested in https://github.com/scala-js/scala-js/pull/4070#discussion_r472058502

Reply:
I think you changed it in `WrappedMapTest` and `WrappedSetTest`, but not in `MapTest` and `SetTest`.